etl/data_helpers/misc.py

```diff
@@ -119,4 +119,36 @@ def add_origins_to_energy_table(tb_energy: Table) -> Table:
     return tb_energy
 
 
+########################################################################################################################
+# TODO: Remote this temporary function once WDI has origins.
+def add_origins_to_mortality_database(tb_who: Table) -> Table:
+    tb_who = tb_who.copy()
+
+    # List all non-index columns in the WDI table.
+    data_columns = [column for column in tb_who.columns if column not in ["country", "year"]]
+
+    # For each indicator, add an origin (using information from the old source) and then remove the source.
+    for column in data_columns:
+        tb_who[column].metadata.sources = []
+        error = "Remove temporary solution where origins where manually created."
+        assert tb_who[column].metadata.origins == [], error
+        tb_who[column].metadata.origins = [
+            Origin(
+                title="Mortality Database",
+                producer="World Health Organisation",
```

Comment:
If you want to change what's shown by default in the footer of charts, you could add here:
```
attribution="WHO Mortality Database (2023)"
```
or `World Health Organisation - Mortality Database (2023)` (whatever we use elsewhere).